src/System.ComponentModel.Composition/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp2.0;
```

Comment:
Please move netcoreapp2.0 into a PackageConfiguration and then include that property into the build configurations. Also into BuildConfigurations we need to add netcoreapp for the vertical build from source.

I will look something like:
```xml
<PropertyGroup>
  <PackageConfigurations>
     netcoreapp2.0;
  </PackageConfigurations>
  <BuildConfigurations>
     $(PackageConfigurations);
     netcoreapp;
     uap;
  </BuildConfigurations>
</PropertyGroup>
```